### PR TITLE
KviKvs*.cpp: typos and consistency fixes

### PR DESCRIPTION
**Pull request by un1versal**
die ellipses die

Please dont let this fester here for long, there's plenty to do still.

@blindsighttf2 @VFiber please have a look

### CODE REVIEW

**Comment by staticfox** [comment]
I should have time to review this tomorrow or Monday.

File: src/kvirc/kvs/KviKvsTimerManager.cpp [hunk 1]
```diff
@@ -202,7 +202,7 @@ void KviKvsTimerManager::timerEvent(QTimerEvent *e)
 	KviKvsTimer * t = m_pTimerDictById->find(iId);
 	if(!t)
 	{
-		qDebug("Internal error: got an nonexistant timer event");
+		qDebug("Internal error: Received a non existent timer event");
```
Comment (ctrlaltca) [inline src/kvirc/kvs/KviKvsTimerManager.cpp]: Isn't the "R" in "received" supposed to be lowercase as the "t" in "trying" in this same commit?

**Comment by ctrlaltca** [comment]
Fine for me :+1:

File: src/kvirc/kvs/KviKvsTimerManager.cpp (hunk 1 above)
Comment (un1versal) [inline src/kvirc/kvs/KviKvsTimerManager.cpp]: yes :) well spoted.

File: src/kvirc/kvs/KviKvsCoreCallbackCommands.cpp [hunk 2]
```diff
@@ -729,7 +729,7 @@ namespace KviKvsCoreCallbackCommands
 			With the -b switch the events stdout and stderr are triggered once even if the process
 			emits no output.
 			The -s=<interpreter> switch may be used to specify the path of the command interpreter
-			that is "sh -c" by default on unix machines and "cmd.exe /c" on windows.
+			that is "sh -c" by default on UNIX machines and "cmd.exe /c" on windows.
```
Comment (staticfox) [inline src/kvirc/kvs/KviKvsCoreCallbackCommands.cpp]: `Windows`

File: src/kvirc/kvs/parser/KviKvsParser.cpp [hunk 3]
```diff
@@ -1560,7 +1560,7 @@ KviKvsTreeNodeInstruction * KviKvsParser::parseAsParameter(const QChar * pBuffer
 		[p]
 		The configuration callback will be invoked when the user will try to configure
 		your addon from the addon management dialog. This callback is useful
-		mainly for complexier graphical scripts that can show up a dialog
+		mainly for complexer graphical scripts that can show up a dialog
```
Comment (staticfox) [inline src/kvirc/kvs/parser/KviKvsParser.cpp]: more complex

**Pull request merged**